### PR TITLE
Broken Link sap.m.routing.Targets

### PR DESCRIPTION
**Pull request by ibibhu**
https://openui5.hana.ondemand.com/#docs/api/symbols/sap.m.routing.Targets.html

### CODE REVIEW

**Comment by flovogt** [comment]
@ibibhu Thanks a lot for your PR. We'll fix the issue.

File: docs/Step_10_Implement_Lazy_Loading_cdab0a1.md [hunk 1]
```diff
@@ -247,5 +247,5 @@ Try it out yourself: Open the *Network* tab of your browser's developer tools an
 **Related Information**  
 
 
-[API Reference: `ap.m.routing.Targets`](https://openui5.hana.ondemand.com/#docs/api/symbols/ap.m.routing.Targets.html)
+[API Reference: `ap.m.routing.Targets`](https://openui5.hana.ondemand.com/#docs/api/symbols/sap.m.routing.Targets.html)
```
Comment (OlMue) [inline docs/Step_10_Implement_Lazy_Loading_cdab0a1.md:250]: [API Reference: `sap.m.routing.Targets`](https://openui5.hana.ondemand.com/#docs/api/symbols/sap.m.routing.Targets.html)

**Review by OlMue — APPROVED** [review]
Added another missing "s" in the link text

**Comment by OlMue** [comment]
Added the correction to the CMS, will be fixed with 1.99. Thanks for your contribution! :-)

**Pull request merged**